server/graphql-schema.js

```diff
@@ -9,6 +9,9 @@ const graphqlSchema = gql`
     """
     scalar Timestamp
 
+    # Required for passing free form Object data
+    scalar Object
```

Comment:
I added the support for this as we're still unsure what exactly we'll be getting back and using from the harness and how much it may differ from the previous implementation. For that reason, I'm expecting the information that will updated on `TestPlanRun.testResults` will continue to be free-form data until that's locked in. I suppose at that point, we can create the proper types here?

Reply:
Introduced temporary types. Need to document their usage.